pkg/utils/utils_test.go

```diff
@@ -390,6 +390,34 @@ func TestGetPodIdentifier(t *testing.T) {
 	}
 }
 
+func TestGetDotPodIdentifier(t *testing.T) {
+	type args struct {
+		podName      string
+		podNamespace string
+	}
+
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: "Valid pod name and namespace",
+			args: args{
+				podName:      "my.pod.name-udp-748dc8d996-fb8b2",
+				podNamespace: "default",
+			},
+			want: "my_pod_name-udp-748dc8d996-default",
+		},
+	}
+	for _, tt := range tests {
```

Comment:
nit: for completeness, let's do a negative test. So add a pod name without and "." and validate that it is not changed

Reply:
Will do

Reply:
Actually, this already existing test for `GetPodIdentifier` should be sufficient as a negative test: https://github.com/aws/aws-network-policy-agent/blob/main/pkg/utils/utils_test.go#L365